src/app/components/elements/modals/inequality/InequalityModal.tsx

```diff
@@ -1,26 +1,14 @@
-import React, {
-  FormEvent,
-  useCallback,
-  useContext,
-  useEffect,
-  useLayoutEffect,
-  useMemo,
-  useRef,
-  useState,
-} from "react";
+import React, { useCallback, useContext, useEffect, useLayoutEffect, useMemo, useRef, useState } from "react";
 import { Inequality, WidgetSpec } from "inequality";
 import { isDefined, parsePseudoSymbolicAvailableSymbols, sanitiseInequalityState } from "../../../../services";
 import { IsaacContentValueOrChildren } from "../../../content/IsaacContentValueOrChildren";
 import { ContentDTO } from "../../../../../IsaacApiTypes";
-import { Input } from "reactstrap";
 import classNames from "classnames";
 import { Markup } from "../../markup";
-import { CHEMICAL_ELEMENTS, EditorMode, LogicSyntax, MenuItemProps, MenuItems } from "./constants";
```

Comment:
It looks like `CHEMICAL_ELEMENTS` is no longer used anywhere else, so it should also be removed from `constants.ts`.